index.html

```diff
@@ -0,0 +1,140 @@
+        <h1>Handcrafed, home-made masterpieces</h1>
+        <form action="https://www.freecodecamp.com/email-submit        ">
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="Put your adresse email"
+            required
+          /><br />
+          <button>GET STARTED</button>
+        </form>
+        <br />
+      </section>
+
+      <div class="Content-center">
+        <section id="features">
+          <div class="grid">
+            <div class="icon"><i class="fas fa-fire" id="icon"></i></div>
+            <div class="description">
+              <h2>Premium Materials</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+          <div class="grid">
+            <div class="icon">
+              <i class="fas fa-truck-moving" id="icon"> </i>
+            </div>
+            <div class="description">
+              <h2>Fast Shipping</h2>
+              <p>
+                We make sure you recieve your trombone as soon as we have
+                finished making it. We also provide free returns if you are not
+                satisfied
+              </p>
+            </div>
+          </div>
+          <div class="grid">
+            <div class="icon">
+              <i class="fas fa-battery-full" id="icon"> </i>
+            </div>
+            <div class="description">
+              <h2>Quality assurance</h2>
+              <p>
+                For every purchase you make, we will ensure there are no damages
+                or faults and we will check and test the pitch of your
+                instrument.
+              </p>
+            </div>
+          </div>
+        </section>
+        <br /><br />
+        <section id="how-it-works">
+          <iframe
+            src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&controls=0&showinfo=0
+                    "
+            frameborder="0"
+          ></iframe>
+        </section>
+        <br /><br /><br />
+        <section id="pricing">
+          <div class="tenor">
+            <div class="trombone">TENOR TROMBONE</div>
+            <br />
+            <h2>600$</h2>
+            <p>Lorem ipsum.</p>
+            <p>Lorem ipsum.</p>
+            <p>Lorem ipsum dolor.</p>
+            <p>Lorem ipsum.</p>
+            <button id="btn">SELECT</button>
+          </div>
+          <div class="tenor">
+            <div class="trombone">BASS TROMBONE</div>
+            <br />
+            <h2>900$</h2>
+            <p>Lorem ipsum.</p>
+            <p>Lorem ipsum.</p>
+            <p>Lorem ipsum dolor.</p>
+            <p>Lorem ipsum.</p>
+            <button id="btn">SELECT</button>
+          </div>
+          <div class="tenor">
+            <div class="trombone">VALVE TROMBONE</div>
+            <br />
```

Comment:
Please, try to use one <p> tag and insert <br> tags inside instead of using many <p> for html best pratice